### ISSUE TITLE
300W Dataset Numbers

### ISSUE THREAD

**Issue by abhi1kumar** [HRNet/HRNet-Facial-Landmark-Detection#16]
Hi authors,
Thankyou for releasing your code. I just happened to see a different number on the README as well as on the arxiv paper for 300W dataset (although the number for the test is same). It would be really great if you could tell the differences between the training of these two models.

 ||Common | Challenge | Full| Test|
 |------------- | ------------- |------------- | ------------- |------------- |
|README|2.91| 5.11 | 3.34 | 3.85|
|Arxiv paper | 2.87 |5.15| 3.32|3.85|

**Comment by wondervictor** [comment]
Hi, our released model is retrained, which is slightly different from the model we used in the paper.

**Comment by zhao181** [comment]
How can you obtain such a result？ I retrain the model with this code，the ocular-distance normalized error is 0.0345 and the pupil-distance normalized error is 0.0489. Can you give some adivices on improving the result？

**Comment by fengyuentau** [comment]
@zhao181 The `nme` shown either from `README.md` or the paper is a percentage, check this paper [Face Alignment at 3000 FPS via Regressing Local Binary Features](http://www.jiansun.org/papers/CVPR14_FaceAlignment.pdf) for more details.

To summary, almost every paper which tests on 300W dataset follows the settings made by this paper. And the following is a section declaring that the numbers (nme) are percentage, and they drop `%` for simplicity.

![image](https://user-images.githubusercontent.com/17219438/84486770-6cfe4680-acd0-11ea-97ff-82e61a2c2a42.png)

Also note that my evaluation results on 300W dataset using the pre-trained model are slightly different from either `README.md` and the paper.:
```shell
common: 2.84, challenge: 5.17, full: 3.38, test: 3.85
```
